node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/nodeagent/NodeAgentImpl.java

```diff
@@ -595,13 +595,22 @@ private void pushMetricsToContainer(NodeAgentContext context, List<DimensionMetr
             String wrappedMetrics = "s:" + params.toString();
 
             // Push metrics to the metrics proxy in each container
-            String[] command = {"vespa-rpc-invoke",  "-t", "2",  "tcp/localhost:19091",  "setExtraMetrics", wrappedMetrics};
-            dockerOperations.executeCommandInContainerAsRoot(context, 5L, command);
+            runPushMetricsCommand(context, wrappedMetrics, true);
```

Comment:
Wont this command fail when run inside a container that does not have the new metrics proxy? F.ex. 6.330?

Reply:
Ok, so it does fail, but it looks like this method does not throw on exit code != 0, so should be ok...